apps/election-manager/src/App.test.tsx

```diff
@@ -335,7 +335,6 @@ it('tabulating CVRs with SEMS file', async () => {
   const totalRow = domGetByTestId(ballotsByVotingMethod[0], 'total')
   domGetByText(totalRow, 'Total Ballots Cast')
   domGetByText(totalRow, '200')
-  expect(getByTestId('tally-report-contents')).toMatchSnapshot()
```

Comment:
same comment --> keep until we have alternative? I don't think the snapshot on the report is all that bad, I don't think it will change willy-nilly with unrelated changes.

Reply:
What is this line testing for? How does the dev know if what changed was supposed to change?

Without a clear assertion the value of this test is not discernible.

Reply:
Overall this repo does not yet have 100% test coverage and when we do, we should not rely on snapshots to blanket cover code which we should be looking for specific values or which have comments defining what the snapshot is checking for such that future devs don't have to figure out what the test is for each time they run in to it. With large snapshots, I fear that a dev will simply update the snapshot and move on (I have! 😛). 

I will add these testing lines back, because there is also no harm in doing so… but with comments noting that large snapshots without clear understanding of what is being checked for are not useful.